client/src/panels/panel_declarations/results/result_components.yaml

```diff
@@ -28,4 +28,9 @@ indicator_display_desc:
 
 report:
   en: Report
-  fr: Rapport
\ No newline at end of file
+  fr: Rapport
+
+result_late_depts_warning:
+  transform: [handlebars,markdown]
+  en: Totals below are incomplete, data is not yet available for **{{late_dept_count}}** {{plural_branch late_dept_count "organization" "organizations"}}. Updates will follow.
+  fr: Les totaux ci-dessous sont incomplets, les données ne sont pas encore disponibles pour **{{late_dept_count}}** {{plural_branch late_dept_count "organisation" "organisations"}}. Des mises à jour suivront.
```

Comment:
Is there a reason this has different text than the other alert?

Are there other panels that should be showing this?

Reply:
This one is gov level only, but, unlike the other gov level warning, I didn't want it displaying the full length list of late departments (since that's already at the top of the page and in footnotes). This basically just exists so users jumping straight to a gov level panel from the home page are made aware when departments are late with their results.

Arguably, the same banner could go on the gov level results intro panel, but nothing (of ours) links directly to that and it's right underneath the main, full length, warning banner.

Dept level (and below) panels don't need it I figure, they aren't actively misleading when a department is late (aside from maybe implying that subject WON'T have that year's doc) and aren't linked to directly so users seeing them should will have scrolled past the main warning banner on the way there (again, not directly linked to by us at least).